src/Helpers/ClientService.php

```diff
@@ -4,32 +4,19 @@
 
 use AcquiaCloudApi\Connector\Client;
 use AcquiaCloudApi\Connector\Connector;
-use Webmozart\KeyValueStore\JsonFileStore;
 
 class ClientService {
 
-  private $acquiaCloudClient;
+  private $connector;
 
-  private $cloud_api_conf;
-
-  public function __construct(JsonFileStore $datastoreCloud) {
-    $this->cloud_api_conf = $datastoreCloud;
+  public function __construct(Connector $connector) {
```

Comment:
Connector is passed automatically via dependency injection, thanks to the config/services.yml fix above